### PR TITLE
Npm release

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:*

*Description of changes:*
Preparation for release.
Cleaned up packages for release, moved some files around, updated CDK version, removed vulnerabilities, reduced package size, updated library versions, fixed code to work with newer versions. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kcoleman731 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -11,43 +11,43 @@
...
+        "@aws-cdk/aws-kms": "1.124.0",
+        "@aws-cdk/aws-logs": "1.124.0",
+        "@aws-cdk/aws-s3": "1.124.0", 
+        "@aws-cdk/aws-secretsmanager": "1.124.0",
+        "@aws-cdk/aws-route53": "1.124.0", 
+        "@aws-cdk/core": "1.124.0",
+        "@aws-cdk/pipelines": "1.124.0",
         "@types/bcrypt": "^5.0.0",
-        "aws-sdk": "^2.952.0",
+        "aws-sdk": "2.996.0",
```
Comment (askulkarni2) [inline package.json:47]: Is there whitespace here after `,`?

**Review by askulkarni2 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (shapirov103) [reply package.json]: need to set vscode to removing trailing whitespace. :)
addressed

**Review by askulkarni2 — APPROVED** [review]
:shipit:

**Pull request merged**